dotnet/DDB.Bindings/DroneDB.cs

```diff
@@ -116,20 +185,42 @@ public static List<Entry> Info(string[] paths, bool recursive = false, int maxRe
 
         public static List<Entry> List(string ddbPath, string path, bool recursive = false, int maxRecursionDepth = 0)
         {
+            if (string.IsNullOrWhiteSpace(path))
+                throw new DDBException("Path should not be null or empty");
+
             return List(ddbPath, new[] { path }, recursive, maxRecursionDepth);
         }
 
         public static List<Entry> List(string ddbPath, string[] paths, bool recursive = false, int maxRecursionDepth = 0)
         {
-            if (_List(ddbPath, paths, paths.Length, out var output, "json", recursive, maxRecursionDepth) !=
-                DDBError.DDBERR_NONE) throw new DDBException(GetLastError());
+            if (string.IsNullOrWhiteSpace(ddbPath))
+                throw new DDBException("DDB path should not be null or empty");
+
+            if (paths == null || paths.Any(string.IsNullOrWhiteSpace))
+                throw new DDBException("One of the provided paths is null or empty");
+
+            try
+            {
+
+                // If the paths are not absolute let's rebase them on ddbPath
+                paths = (from path in paths
```

Comment:
Mm, I somehow feel this logic should be part of Registry, not the bindings.

Reply:
See https://github.com/DroneDB/DroneDB/pull/115#issuecomment-709561637